src/main/java/io/temporal/failure/ApplicationFailure.java

```diff
@@ -51,49 +51,47 @@
  */
 public final class ApplicationFailure extends TemporalFailure {
   private final String type;
-  private final Value details;
+  private final Values details;
   private boolean nonRetryable;
 
   /**
+   * New ApplicationFailure with {@link #isNonRetryable()} flag set to false. Note that this
```

Comment:
didn't quite follow the 2nd sentence of this comment

Reply:
Good catch.